cg/server/api.py

```diff
@@ -67,7 +60,8 @@
 @BLUEPRINT.route("/submit_order/<order_type>", methods=["POST"])
 def submit_order(order_type):
     """Submit an order for samples."""
-    api = OrdersAPI(lims=lims, status=db, osticket=osticket)
+
+    api = orders_api
```

Comment:
This can be removed, no need to rename the `orders_api` that has been imported

Reply:
👍